api/OTPCredential.json

```
@@ -0,0 +1,98 @@
+{
+  "api": {
+    "OTPCredential": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": true
```

```suggestion
            "version_added": "84"
```

Based on https://storage.googleapis.com/chromium-find-releases-static/f75.html#f75589ed72c4019f8903256fb2894f5503d95ca0

Can you mirror the rest of the data to match?